### PR TITLE
doc: Update README and link to bootloader docs

### PR DESCRIPTION
**Pull request by jeez**
Signed-off-by: Jesus Sanchez-Palencia jesus.sanchez-palencia@intel.com

### CODE REVIEW

**Comment by jeez** [comment]
@dalessan after this, I think we can tag and package the release here. Should we add binaries to the release page?

File: README.rst [hunk 1]
```diff
@@ -9,6 +9,9 @@ It ranges from small devices like micro-controller boards up to mobile phones.
 *qm-dfu-util* allows you to download firmware to your Intel® Quark™
 Microcontroller by using the serial interface (UART).
 
+For more information about this functionality and how to interface with the
+QMSI bootloader, please refer to https://github.com/quark-mcu/qmsi#device-firmware-update-dfu .
```
Comment (dalessan) [inline README.rst]: Why not linking the guide directly?
https://github.com/quark-mcu/qmsi/blob/master/doc/dfu.rst

Comment (jeez) [inline README.rst]: fine by me

**Comment by dalessan** [comment]
LGTM

**Pull request merged**